src/ui/hash.js

```diff
@@ -15,9 +15,12 @@ import type Map from './map';
 class Hash {
     _map: Map;
     _updateHash: () => ?TimeoutID;
+    _hashName: ?string;
 
-    constructor() {
+    constructor(hashName: string | boolean) {
```

Comment:
What does `hashName` of boolean type do here? Shouldn't the `hashName` be an optional string parameter

Reply:
Exact, I updated `map.js` in consequence